### PR TITLE
daniel/UpdateFacebookFriends

### PR DESCRIPTION
**Pull request by DanielNosrati**
Get friends pagination and tests. Style changes
All tests passing.

### CODE REVIEW

File: src/app/pcasts/controllers/get_facebook_friends.py [hunk 1]
```diff
@@ -8,18 +8,21 @@ def get_path(self):
     return '/users/facebook/friends/'
```
Comment (kevingreer) [inline src/app/pcasts/controllers/get_facebook_friends.py:8]: should update the apiary to include this endpoint

**Review by kevingreer — APPROVED** [review]
Looks good

**Comment by DanielNosrati** [comment]
The dao fle was patched wrongly because i was working on search and friends on the same branch. The commit removes a bad function which was not caught because this PR and search were tested jointly.

**Comment by kevingreer** [comment]
I'm not sure I follow what went wrong, you said all the tests were passing but on the github slack you said you pushed without testing? Was it due to the order you merged stuff?

**Comment by DanielNosrati** [comment]
Yes. I was working on the two features on the same branch because they were both dealing with the same errors. I then "git add --patched" the dao file(letting you choose what parts of the file to commit) which was modified by both the search feature and the friends feature. I rejected the hunk that deleted the old function so the overloaded function was still there(in the git index) even though it wasn't anymore. I ran tests for this patch with both the search feature and the friends feature which passed since the search feature didn't have the overloaded function that caused the tests to fail.

**Pull request merged**